bluetooth/src/commonMain/kotlin/device/DeviceState.kt

```diff
@@ -63,14 +67,18 @@ sealed class DeviceAction {
          * @param newValue the [ByteArray] to write
          * @property characteristic the [com.splendo.kaluga.bluetooth.Characteristic] to read the value of
          */
-        class Characteristic(newValue: ByteArray, val characteristic: com.splendo.kaluga.bluetooth.Characteristic) : Write(newValue)
+        class Characteristic(newValue: ByteArray, val characteristic: com.splendo.kaluga.bluetooth.Characteristic) : Write(newValue) {
+            override fun toString(): String = "DeviceAction.Write.Characteristic(${characteristic.uuid})"
```

Comment:
could have added `newValue` into the printout, but it didn't look right to me. besides there are other places printing the applied values